next.config.js

```diff
@@ -0,0 +1,44 @@
+let config = {
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
+    // your project has type errors.
+    // !! WARN !!
+    ignoreBuildErrors: true
+  },
+  async rewrites() {
+    return [
+      {
+        source: "/api/configs_db/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/config/db/:path*"
```

Comment:
Do these not need to come from the environment variables ?

Reply:
Ya, they can. Just porting the existing `_redirects` netlify config.